OpenSim/Simulation/Model/GeometryPath.cpp

```diff
@@ -350,14 +350,14 @@ void GeometryPath::addInEquivalentForces(const SimTK::State& s,
             // add in the tension point forces to body forces
             if (mppo) {// moving path point location is a function of the state
                 // transform of the frame of the point to the base mobilized body
-                auto X_BF = mppo->getParentFrame().findTransformInBaseFrame();
+                auto X_BF = mppo->getBody().findTransformInBaseFrame();
```

Comment:
This was a mistake on my part. I only replaced to verify that `getBody()` still works and meant to revert and not commit the change. I took a pass through GeometryPath to update all the `Body` related calls to `ParentFrame`.